tests/test_writer.py

```diff
@@ -1,50 +1,56 @@
-from pytest import fixture
+from pytest import yield_fixture
 
 from pip.index import FormatControl
-from piptools.utils import comment
+from piptools.utils import comment, UNSAFE_PACKAGES
 from piptools.writer import OutputWriter
 
 
-@fixture
+class Writer(object):
```

Comment:
By yielding a class that I can invoke later I am allowing for keyword arguments to be passed in so that I can reuse this fixture while setting `allow_unsafe`.

I'll update the fixture annotation